www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/reverse_proxy.volt

```diff
@@ -300,7 +319,7 @@
         <!-- Button group on the left -->
         <div>
             <button id="addDomainBtn" type="button" class="btn btn-secondary">{{ lang._('Step 1: Add Domain') }}</button>
-            <button id="addHandleBtn" type="button" class="btn btn-secondary">{{ lang._('Step 2: Add Upstream') }}</button>
+            <button id="addHandleBtn" type="button" class="btn btn-secondary">{{ lang._('Step 2: Add HTTP Handler') }}</button>
```

Comment:
Does this require docs changes?

Reply:
Yes it does, there is a PR in the doc section.
https://github.com/opnsense/docs/pull/581